core/sail/api/src/main/java/org/eclipse/rdf4j/sail/helpers/SailConnectionWrapper.java

```diff
@@ -111,6 +111,12 @@ public void close() throws SailException {
 		return wrappedCon.getStatements(subj, pred, obj, includeInferred, contexts);
 	}
 
+	@Override
+	public boolean hasStatement(Resource subj, IRI pred, Value obj, boolean includeInferred, Resource... contexts)
+			throws SailException {
+		return wrappedCon.hasStatement(subj, pred, obj, includeInferred, contexts);
+	}
+
```

Comment:
Not really related, but: good catch.

Reply:
I actually branched from the wrong place and brought this along. It has its own PR.